doc/stdlib/io-network.rst

```diff
@@ -900,53 +873,53 @@ Memory-mapped I/O
 Network I/O
 -----------
 
-.. function:: connect([host],port) -> TCPSocket
+.. function:: connect([host], port::Integer) -> TCPSocket
 
    .. Docstring generated from Julia source
 
    Connect to the host ``host`` on port ``port``\ .
 
-.. function:: connect(path) -> PipeEndpoint
+.. function:: connect(path::AbstractString) -> PipeEndpoint
 
    .. Docstring generated from Julia source
 
    Connect to the named pipe / UNIX domain socket at ``path``\ .
 
-.. function:: listen([addr,]port) -> TCPServer
+.. function:: listen([addr, ]port::Integer; backlog::Integer=BACKLOG_DEFAULT) -> TCPServer
```

Comment:
It's how many connections can be pending (not having called accept) before the server starts dropping them on the floor.